superset-frontend/src/SqlLab/components/QueryAutoRefresh/QueryAutoRefresh.test.tsx

```diff
@@ -34,6 +36,22 @@ import { QueryDictionary } from 'src/SqlLab/types';
 
 const middlewares = [thunk];
 const mockStore = configureStore(middlewares);
+const mockState = {
```

Comment:
NOTE: could be good to put in a reusable fixture file if there's a good place for it.